backend/spec/fixtures/vcr_cassettes/calculate_project_impact_score.json

```diff
@@ -0,0 +1,1688 @@
+{
```

Comment:
I have used this file to replace `backend/spec/fixtures/vcr_cassettes/export.json` which had more then 17.000 lines -- this one is 10x shorter, which is probably caused by decreasing resolution of `aries.heco.locations.colombia_continental`, which makes BC3 response a lot faster...